### ISSUE TITLE
assertion failed in llvm_emit_const_initializer

### ISSUE THREAD

**Issue by pierrec** [c3lang/c3c#786]
```
module game;
import rl;

struct Rec2
{
    Color color;
}

fn void main(String[] args)
{
    Rec2 rec = { .color = rl::GRAY };
}
```

where rl is [Raylib.c3](https://github.com/Its-Kenta/Raylib.c3)
produces the following error:
> Assertion failed: (LLVMIsConstant(element)), function llvm_emit_const_initializer, file llvm_codegen.c, line 278.

**Comment by lerno** [comment]
Awesome, thank you, I think I know exactly the problem.

**Comment by lerno** [comment]
This is fixed in 0.4.532
